01-JS-DOM-APIs/script.js

```diff
@@ -0,0 +1,120 @@
+// exercise 6
+function show(){
+    let obj = document.getElementById("hidden");
+    obj.style.visibility ="visible";
```

Comment:
As we discussed, try to define a class with your visible styles, and add or remove that class to obj :)
Also, try to name your variables in a more descriptive way, something like *hiddenSection*.